### PR TITLE
Add support for installing instance as "NT AUTHORITY\LocalService"

### PR DESCRIPTION
**Pull request by johnsimons**
Connects to #715

### CODE REVIEW

**Comment by gbiellem** [comment]
@johnsimons  I've fixed up the LocalService option so it works regardless of the locale of the OS.  Tested on English Windows 10 and German Windows 8.1

**Comment by johnsimons** [comment]
So is there anything left to do @gbiellem ?

File: src/ServiceControlInstaller.Engine/Services/WindowsServiceController.cs [hunk 1]
```diff
@@ -107,15 +107,19 @@ public void ChangeAccountDetails(string serviceAccount, string servicePassword)
                     var message = (wmiReturnCode < Win32ChangeErrorMessages.Length)
                         ? $"Failed to change service credentials on service {ServiceName} - {Win32ChangeErrorMessages[wmiReturnCode]}"
                         : "An unknown error occurred";
+
+                    if (wmiReturnCode == 22)
+                    {
+                        message += $"( AccountName {userAccount.QualifiedName} converted to {ConvertAccountNameToServiceAccount(userAccount.QualifiedName)})";
```
Comment (johnsimons) [inline src/ServiceControlInstaller.Engine/Services/WindowsServiceController.cs]: @gbiellem is this a debug statement?

**Comment by johnsimons** [comment]
looks good, but one comment added

**Comment by gbiellem** [comment]
I left that in on purpose - on the off chance we do hit a locale issue that this fix cover

**Pull request merged**